yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -59,23 +59,24 @@ func prepareDebeziumConfig(partitionsToRootTableMap map[string]string, tableList
 	default:
 		return nil, nil, fmt.Errorf("invalid export type %s", exportType)
 	}
-	fmt.Printf("num tables to export: %d\n", len(tableList))
-	utils.PrintAndLog("table list for data export: %v", tableList)
 	tableNameToApproxRowCountMap := getTableNameToApproxRowCountMap(tableList)
 
 	var dbzmTableList, dbzmColumnList []string
 	for _, table := range tableList {
+		t := table.Qualified.MinQuoted
+		if source.DBType == POSTGRESQL && table.SchemaName.MinQuoted == "public" {
+			t = table.ObjectName.MinQuoted
+		}
+		if leafPartitions[t] != nil {
+			//In case of debezium offline migration of PG, tablelist should not have root and leaf both so not adding root table in table list
```

Comment:
just checking, this flow will kick in even for live migration, correct?

Reply:
Yes right, but atleast even if we pass table-list with root to dbzm there won't be any issue as events only be exported on the basis of leaf partitions but in snapshot phase we can't as you were also mentioning that it duplicates the data for parititons and root.